microprofile/server/src/main/java/io/helidon/microprofile/server/ServerCdiExtension.java

```diff
@@ -117,6 +126,9 @@ private void startServer(@Observes @Priority(PLATFORM_AFTER + 100) @Initialized(
         // register static content if configured
         registerStaticContent();
 
+        // register websocket endpoints
```

Comment:
Extract into a separate module, use `WebSocketCdiExtension` in that module to setup everything